plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -196,11 +218,32 @@ const addToCartWithOptionsStore = store<
 				}
 
 				if ( productType === 'grouped' ) {
-					return Object.values( quantity ).some( ( qty ) => qty > 0 );
+					return Object.values( quantity ).some(
+						( itemQty ) => itemQty > 0
+					);
 				}
 
 				return true;
 			},
+			get variationId(): number | null {
+				const context = getContext< Context >();
+				if ( ! context ) {
+					return null;
+				}
+				const { availableVariations, selectedAttributes } = context;
+				const matchedVariation = getMatchedVariation(
+					availableVariations,
+					selectedAttributes
+				);
+				return matchedVariation?.variation_id || null;
+			},
+			get selectedAttributes(): SelectedAttributes[] {
+				const context = getContext< Context >();
+				if ( ! context ) {
+					return [];
+				}
+				return context.selectedAttributes;
+			},
```

Comment:
You shouldn't need to add this. These states are already added to the same store in this file:

https://github.com/woocommerce/woocommerce/blob/32cbb108b33002991e722437a7986f847ad974c2/plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/frontend.ts#L172-L190

(It's two different files, but they both add states/actions/callbacks to the same store. This way we only load the variable product specific things when the product is, in fact, a variable product)